highfidelity/hifi
# Fix toolbar apps switch

This PR fixes 2 cases:
- https://highfidelity.fogbugz.com/f/cases/6327/Switching-from-Create-Toolbar-App-Closes-Dialog-automatically
- https://highfidelity.fogbugz.com/f/cases/6551/QT591-User-is-unable-to-change-to-certain-tabs-from-GOTO

Test plan:
- Switch to Toolbar mode
- open Create app
- Switch to other apps
- Make sure window with the app remains opened and new app appears in the window
- Click on same app button again.
- Make sure window is closed
- open Go To app
- do the same thing to make sure you can go from app to app
